### PR TITLE
NB for examples

### PR DESCRIPTION
**Pull request by bethz**
These are the sample NB to showcase the MLOps Run Jupyter Notebook extension

### CODE REVIEW

File: notebooks/dkn_synthetic.ipynb [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "sys.path.append(\"../../\")\n",
```
Comment (emcmanu) [inline notebooks/dkn_synthetic.ipynb]:
```suggestion
    "sys.path.append(\"./\")\n",
```

File: notebooks/ncf_movielens.ipynb [hunk 2]
```diff
@@ -0,0 +1,381 @@
...
+      "System version: 3.6.7 | packaged by conda-forge | (default, Nov 21 2018, 03:09:43) \n",
+      "[GCC 7.3.0]\n",
+      "Pandas version: 0.23.4\n",
+      "Tensorflow version: 1.12.0\n"
+     ]
+    }
+   ],
+   "source": [
+    "import sys\n",
+    "sys.path.append(\"../../\")\n",
```
Comment (emcmanu) [inline notebooks/ncf_movielens.ipynb]:
```suggestion
    "sys.path.append(\"./\")\n",
```

File: notebooks/fastai_movielens.ipynb [hunk 3]
```diff
@@ -0,0 +1,830 @@
...
+      "Torch version: 1.0.0\n",
+      "Cuda Available: True\n",
+      "CuDNN Enabled: True\n"
+     ]
+    }
+   ],
+   "source": [
+    "# set the environment path to find Recommenders\n",
+    "import sys\n",
+    "sys.path.append(\"../../\")\n",
```
Comment (emcmanu) [inline notebooks/fastai_movielens.ipynb]:
```suggestion
    "sys.path.append(\"./\")\n",
```

File: notebooks/rbm_movielens.ipynb [hunk 4]
```diff
@@ -0,0 +1,823 @@
...
+   "source": [
+    "#load libraries\n",
+    "\n",
+    "from __future__ import print_function\n",
+    "from __future__ import absolute_import\n",
+    "from __future__ import division\n",
+    "\n",
+    "# set the environment path to find Recommenders\n",
+    "import sys\n",
+    "sys.path.append(\"../../\")\n",
```
Comment (emcmanu) [inline notebooks/rbm_movielens.ipynb]:
```suggestion
    "sys.path.append(\"./\")\n",
```

File: notebooks/sar_movielens.ipynb [hunk 5]
```diff
@@ -0,0 +1,595 @@
...
+     "text": [
+      "System version: 3.6.8 |Anaconda, Inc.| (default, Feb 11 2019, 15:03:47) [MSC v.1915 64 bit (AMD64)]\n",
+      "Pandas version: 0.24.1\n"
+     ]
+    }
+   ],
+   "source": [
+    "# set the environment path to find Recommenders\n",
+    "import sys\n",
+    "sys.path.append(\"../../\")\n",
```
Comment (emcmanu) [inline notebooks/sar_movielens.ipynb]:
```suggestion
    "sys.path.append(\"./\")\n",
```

File: notebooks/wide_deep_movielens.ipynb [hunk 6]
```diff
@@ -0,0 +1,1282 @@
...
+     "text": [
+      "Tensorflow Version: 1.12.0\n",
+      "['/device:CPU:0', '/device:XLA_CPU:0', '/device:XLA_GPU:0', '/device:GPU:0']\n",
+      "Num CPUs: 6\n"
+     ]
+    }
+   ],
+   "source": [
+    "import sys\n",
+    "sys.path.append(\"../../\")\n",
```
Comment (emcmanu) [inline notebooks/wide_deep_movielens.ipynb]:
```suggestion
    "sys.path.append(\"./\")\n",
```

File: notebooks/xdeepfm_synthetic.ipynb [hunk 7]
```diff
@@ -0,0 +1,494 @@
...
+     "text": [
+      "System version: 3.6.7 | packaged by conda-forge | (default, Nov 21 2018, 03:09:43) \n",
+      "[GCC 7.3.0]\n",
+      "Tensorflow version: 1.12.0\n"
+     ]
+    }
+   ],
+   "source": [
+    "import sys\n",
+    "sys.path.append(\"../../\")\n",
```
Comment (emcmanu) [inline notebooks/xdeepfm_synthetic.ipynb]:
```suggestion
    "sys.path.append(\"./\")\n",
```

**Comment by praneetmsft** [comment]
All the Notebooks are moved to : https://github.com/Microsoft/MLOpsSamples/tree/master/notebooks
PR is no longer needed.

**Pull request closed without merging**